### DISCUSSION TITLE
When clicking the PageDown and PageUp keys of the Keyboard, SumatraPDF should always show the top of the page.

### DISCUSSION THREAD

**Discussion by NikonZ9H** [sumatrapdfreader/sumatrapdf#4486] [Ideas]
I customized SumatraPDF's Advanced Settings (so it does NOT get changed at all) to have a Setting of 
      DefaultZoom = 124
which displays the page (8.5 inch x 11 inch) in 100% portrait mode on a 34 inch LCD monitor. If the document is multi-page and I press the PageDown or PageUp key, the next page is not displayed at the top of the page. If this is a settings key, I don't know what it is. If this is not a settings key, can the new feature be implemented as an Advanced Settings key?
<img width="786" alt="SumatraPDF_PageDownPageUp_Suggestion_01" src="https://github.com/user-attachments/assets/f4a0e044-f851-4d34-ba55-0abb2659a50e">
<img width="786" alt="SumatraPDF_PageDownPageUp_Suggestion_02" src="https://github.com/user-attachments/assets/4c08f08a-4012-45cd-a17b-e68cbf9a85f9">
<img width="786" alt="SumatraPDF_PageDownPageUp_Suggestion_03" src="https://github.com/user-attachments/assets/f29da2ae-bf74-4cda-8df9-cf8e91d9a1a0">
The above description of the feature is for a PDF in Portrait mode, can a new feature be also implemented such that when the page is in
Landscape mode, the DefaultZoom for Portrait mode automatically re-adjust to fit the entire Landscape mode PDF document just like
the behavior of a Portrait mode PDF page.

**Comment by GitHubRulesOK** [comment]
to scroll to top of next page it is <kbd> **n** </kbd>
to scroll to top of previous page it is <kbd> **p** </kbd>
to best fit page in either orientation is <kbd> **Ctrl** </kbd> + <kbd> **0** </kbd>(zero) and to switch to from fit to continuous scroll mode it is <kbd> **c** </kbd>

These are the defaults as shown in help or https://www.sumatrapdfreader.org/docs/Keyboard-shortcuts
